packages/config/config/devices/0x019b/z-trm6.json

```diff
@@ -0,0 +1,469 @@
+{
+	"manufacturer": "Heatit",
+	"manufacturerId": "0x019b",
+	"label": "Z-TRM6",
+	"description": "Floor thermostat",
+	"devices": [
+		{
+			"productType": "0x0030",
+			"productId": "0x3001",
+			"zwaveAllianceId": 4908
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 1,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Binary Switch Set",
+			"maxNodes": 10
+		},
+		"3": {
+			"label": "Thermostat Setpoint Set",
+			"maxNodes": 10
+		},
+		"4": {
+			"label": "Thermostat Mode Set",
+			"maxNodes": 10
+		}
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"label": "Disable Buttons",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "Enable",
+					"value": 0
+				},
+				{
+					"label": "Disable",
+					"value": 1
+				}
+			]
```

Comment:
We often call this Local Control. Would this make sense here too? In that case, the meaning of Enable and Disable would have to be switched.

Reply:
@geirra I think you missed this one